src/Cache.cpp

```diff
@@ -50,9 +50,9 @@
 static const std::string CURRENT_CACHE_FORMAT_VERSION("2020.10.20");
 static const std::string SECRET("secret");
 
-static lmdb::val NEXT_BATCH_KEY("next_batch");
-static lmdb::val OLM_ACCOUNT_KEY("olm_account");
-static lmdb::val CACHE_FORMAT_VERSION_KEY("cache_format_version");
+static const std::string_view NEXT_BATCH_KEY("next_batch");
```

Comment:
maybe comment this area saying they are lmdbxx variables, since they're not explicitly tied to the lmdb namespace anymore.

Reply:
Hm, maybe, but they are still obviously called key :D